content/python/concepts/pip/pip.md

```diff
@@ -0,0 +1,50 @@
+---
+Title: 'pip'
+Description: 'Much like how npm is used to manage code packages in JavaScript, pip also offers practical ways to install and use pre-made code.'
+Tags:
+  - 'Conceptual'
+  - 'Packages'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/learn-python3'
+---
+
+`pip` is the package installer used in Python to install code packages. A similar tool is Node Package Manager (npm) in JavaScript. Much like npm, `pip` is the same community driven package space, but for Python.
+
+## Packages
+
+The concept of packages is not unique to `pip`. A package is essentially a collection of code that is designed for a specific purpose. One example is the [neuralart](https://pypi.org/project/neuralart/) project, which is a project on Python Package Index (PyPI) that is used for "rendering generative art from a randomly initialized neural network." It's a very cool package that creates computer generated art.
```

Comment:
> A package is essentially a collection of code that is designed for a specific purpose.

In order to engage with as many readers as possible, I would avoid the use of words like: 

* Easy/easily
* Basic(ally)
* Essential(ly)
* Simple/simply

Also, while it is true that packages are designed for a specific purpose, it's important to mention that they are meant to be distributed and reused by other developers in their own projects.

Reply:
> One example is the [neuralart](https://pypi.org/project/neuralart/) project, which is a project on Python Package Index (PyPI) that is used for "rendering generative art from a randomly initialized neural network." It's a very cool package that creates computer generated art.

I would omit the last sentence since the previous sentence already describes what the package is and what it does.